pandas/core/series.py

```diff
@@ -1890,14 +1895,7 @@ def sortlevel(self, level=0, ascending=True, sort_remaining=True):
         -------
         sorted : Series
         """
-        if not isinstance(self.index, MultiIndex):
-            raise TypeError('can only sort by level with a hierarchical index')
-
-        new_index, indexer = self.index.sortlevel(level, ascending=ascending,
-                                                 sort_remaining=sort_remaining)
-        new_values = self.values.take(indexer)
-        return self._constructor(new_values,
-                                 index=new_index).__finalize__(self)
+        return self.sort_index(level=level, ascending=ascending, sort_remaining=sort_remaining)
```

Comment:
Not a comment for this line, but for the docstring a few lines above: can you add a link `sort_index(level=..)`?